bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -79,6 +81,22 @@ trait UtilBijections {
   }
 
   /**
+   * Injection from java futures to twitter futures
+   */
+  implicit def twitter2JavaFuture[A](
```

Comment:
injections only need to go one way.  you can just return None if it's not a ConstFuture.